variants/waveshare_esp32_s3_zero/pins_arduino.h

```diff
@@ -0,0 +1,71 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+#define USB_VID          0x303a
+#define USB_PID          0x822B
+#define USB_MANUFACTURER "Waveshare"
+#define USB_PRODUCT      "ESP32-S3-Zero"
+#define USB_SERIAL       ""  // Empty string for MAC address
+
+// Partial voltage measurement method
+#define WS_RGB 21
+
+// Mapping based on the ESP32S3 data sheet - alternate for OUTPUT
+static const uint8_t OUTPUT_IO1 = 1;
+static const uint8_t OUTPUT_IO2 = 2;
+static const uint8_t OUTPUT_IO3 = 3;
+static const uint8_t OUTPUT_IO4 = 4;
+static const uint8_t OUTPUT_IO5 = 5;
+static const uint8_t OUTPUT_IO6 = 6;
+static const uint8_t OUTPUT_IO7 = 7;
+static const uint8_t OUTPUT_IO8 = 8;
+static const uint8_t OUTPUT_IO9 = 9;
+static const uint8_t OUTPUT_IO10 = 10;
+static const uint8_t OUTPUT_IO11 = 11;
+static const uint8_t OUTPUT_IO12 = 12;
+static const uint8_t OUTPUT_IO13 = 13;
+
+// UART0 pins
+static const uint8_t TX = 43;
+static const uint8_t RX = 44;
+
```

Comment:
I2C and SPI pins definitions are missing, so CI is failing. Please add those.
If you don't want to specify the pins, you can use -1 as the PIN number.